xray_test.go

```diff
@@ -174,3 +174,18 @@ func verifyScanResults(t *testing.T, content string, minViolations, minVulnerabi
 	assert.True(t, len(results[0].Vulnerabilities) >= minVulnerabilities, fmt.Sprintf("Expected at least %d vulnerabilities in scan results, but got %d vulnerabilities.", minVulnerabilities, len(results[0].Vulnerabilities)))
 	assert.True(t, len(results[0].Licenses) >= minLicenses, fmt.Sprintf("Expected at least %d Licenses in scan results, but got %d Licenses.", minLicenses, len(results[0].Licenses)))
 }
+
+func TestXrayCurl(t *testing.T) {
+	initXrayTest(t, commands.GraphScanMinXrayVersion)
```

Comment:
I'm not sure we need the version check, can we just check this?
```go
if !*tests.TestXray {
	t.Skip("Skipping Xray test. To run Xray test add the '-test.xray=true' option.")
}
```
For example, maybe in `initXrayVersion` we can return early when no min Xray version is provided as a parameter.